docs/user/contextplugin.md

```diff
@@ -0,0 +1,19 @@
+# FlowForge Persistent Context Plugin
+
+Projects running within FlowForge include a plugin that make it possible
+to store values in persistent context in a containered environment.
+
+
+### Known Limitations
+
+Currently, only async versions of context `get` and context `set` functions are supported. 
+View the readme for more info.
+
+
+### Requirements
+
```

Comment:
How does a user validate it's available?

Reply:
Good question. 

From a quick visual, the presence of 2 context stores in (for example) a change node indicates the settings have updated the Node-RED settings file because the license is present. 

As for how we know the backend is installed, setup, running, etc I am not sure how to portray this.

Maybe that is covered in the docs/install/quota settings @hardillb  is working on?

Reply:
outdated